### PR TITLE
add ListView and InfoPage content

### PR DESCRIPTION
**Pull request by StefWe**
Add ListView with avilable devices. Add basic navigation between pages and content on the InfoPage

### CODE REVIEW

File: app/qml/pages/InfoPage.qml [hunk 1]
```diff
@@ -8,5 +8,71 @@ Page {
...
+            text: i18n.tr("This application let you syncronize with AsteroidOS Smartwatch")
+        }
+
+        Label {
+            width: parent.width
+            linkColor: UbuntuColors.orange
+            horizontalAlignment: Text.AlignHCenter
+            wrapMode: Text.WrapAtWordBoundaryOrAnywhere
+            //TRANSLATORS: Please make sure the URLs are correct
+            text: i18n.tr("Released under the terms of the <a href=\"https://github.com/AsteroidOS/telescope/LICENSE\">GNU GPL v3</a>")
```
Comment (FlorentRevest) [inline app/qml/pages/InfoPage.qml]: The URL should actually be https://github.com/AsteroidOS/telescope/blob/master/LICENSE

File: app/qml/pages/InfoPage.qml [hunk 2]
```diff
@@ -8,5 +8,71 @@ Page {
...
+            color: UbuntuColors.ash
+            horizontalAlignment: Text.AlignHCenter
+            text: i18n.tr("Version ") + "0.0.2"
+        }
+
+        Label {
+            width: parent.width
+            horizontalAlignment: Text.AlignHCenter
+            wrapMode: Text.WrapAtWordBoundaryOrAnywhere
+            text: i18n.tr("This application let you syncronize with AsteroidOS Smartwatch")
```
Comment (FlorentRevest) [inline app/qml/pages/InfoPage.qml]: synchronize*
"AsteroidOS smartwatches"* or "an AsteroidOS smartwatch"* but not "AsteroidOS Smartwatch"

**Comment by StefWe** [comment]
Thanks for the quick review and your patience.

File: app/qml/pages/InfoPage.qml (hunk 2 above)
Comment (StefWe) [reply app/qml/pages/InfoPage.qml]: i changed it to "AsteroidOS smartwatches"

File: app/qml/pages/InfoPage.qml (hunk 1 above)
Comment (StefWe) [reply app/qml/pages/InfoPage.qml]: done

**Pull request merged**